docs/docsource/Installation/configure/adding_CIFS_storage.rst

```diff
@@ -109,7 +109,7 @@ steps in `Orbit <https://admin.zenko.io>`__:
 
 8. Manually Trigger Sync (optional)
 
-   This chart deploys a Kubernetes CronJob, which periodically launches
+   This chart deploys a Kubernetes cron job, which periodically launches
```

Comment:
Not a big deal but the `CronJob` is a first class object in Kubernetes which is why it was formatted like that.